lib/kube/proxy/utils_testing.go

```diff
@@ -122,7 +122,8 @@ func SetupTestContext(ctx context.Context, t *testing.T, cfg TestConfig) *TestCo
 	require.NoError(t, err)
 	t.Cleanup(func() { require.NoError(t, authServer.Close()) })
 
-	testCtx.TLSServer, err = authServer.NewTestTLSServer()
+	testCtx.TLSServer, err = authServer.NewTestTLSServer(
```

Comment:
Possibly a comment here reiterating the PR description explaining why we're bumping these fields is worthwhile.

Reply:
2a89275fce17c7e7d943679cd1c0637a7855db06